.github/workflows/deploy_branch.yml

```diff
@@ -105,6 +105,23 @@ jobs:
         timeoutSeconds: 720
         intervalSeconds: 15
 
+    - name: Trigger Smoke Test
+      uses: benc-uk/workflow-dispatch@v1.1
+      with:
+        workflow: Smoke Test
+        token: ${{ secrets.PERSONAL_TOKEN }}
+        inputs: '{"smoke_test_domain": "dev.teaching-vacancies.service.gov.uk", "sha": "${{ github.sha }}"}'
```

Comment:
use `${{ env.BRANCH }}` instead?

Reply:
Thanks. Have adjusted that in lines 113 and 120